src/js/components/Tip/Tip.js

```diff
@@ -43,12 +43,20 @@ const Tip = forwardRef(({ children, content, dropProps, plain }, tipRef) => {
     Children.only(children);
 
   const clonedChild = cloneElement(child, {
-    onMouseEnter: () => setOver(true),
-    onMouseLeave: () => setOver(false),
+    onMouseEnter: () => {
+      if (child.props?.onMouseEnter) child.props.onMouseEnter();
```

Comment:
We need to pass the actual `event` object through:
`onMouseEnter: (event) => if (child.props?.onMouseOver) child.props.onMouseEnter(event);`

Reply:
Updated in latest commit